sdarq/frontend/src/app/app-form/form.json

```diff
@@ -0,0 +1,580 @@
+                    "choices": [{
+                            "value": "azure",
+                            "text": "Microsoft Azure"
+                        },
+                        {
+                            "value": "gcp",
+                            "text": "Google Cloud Platform"
+                        },
+                        {
+                            "value": "azure_gcp",
+                            "text": "Both"
+                        }
+                    ]
+                },
+                {
+                    "type": "radiogroup",
+                    "name": "Architecture",
+                    "title": "Your service must be based on one of the following",
+                    "choices": [{
+                            "value": "Managed_kubernetes",
+                            "text": "Managed Kubernetes"
+                        },
+                        {
+                            "value": "Managed_vm",
+                            "text": "Managed VM"
+                        }
+                    ],
+                    "otherText": "Other"
+                }
+            ],
+            "title": "Architecture"
+        },
+        {
+            "name": "Security",
+            "elements": [{
+                    "type": "checkbox",
+                    "name": "Security tools",
+                    "title": "Please select applicable items",
+                    "choices": [{
+                            "value": "dependency_scan",
+                            "text": "We are currently using SourceClear/Snyk or similar tools for downstream dependency/3rd party component management "
+                        },
+                        {
+                            "value": "Git-secrets",
+                            "text": "We are currently using git-secrets in our repository"
+                        },
+                        {
+                            "value": "Secret_management",
+                            "text": "We are currently using tools to store our secrets"
+                        },
+                        {
+                            "value": "Monitoring_tool",
+                            "text": " We are currently using Sentry or a similar tool (ex: Google Debug) for exception handling and reporting"
+                        },
+                        {
+                            "value": "SAST",
+                            "text": "We are currently using Codacy, SonacCloud or similar tools for security static code analysis"
+                        },
+                        {
+                            "value": "Image_scan",
+                            "text": "We are currently using tools to scan our Docker images"
+                        },
+                        {
+                            "value": "DAST",
+                            "text": "Our service is currently scanned weekly/monthly via ZAP or similar tools (if there is a static IP)"
```

Comment:
Should there be an extra related question such as "Our service is deployed to an environment at a static URL"

Reply:
I think, for now, let's keep it like this, we still don't have a lot of information about apps. Maybe we add it in the future.